partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnDateAddMonth.kt

```diff
@@ -65,7 +65,7 @@ internal val Fn_DATE_ADD_MONTH__INT_DATE__DATE = Function.static(
     Datum.date(datetimeValue.plusMonths(intervalValue))
 }
 
-internal val Fn_DATE_ADD_MONTH__INT32_TIMESTAMP__TIMESTAMP = Function.static(
+internal val Fn_DATE_ADD_MONTH__INT32_TIMESTAMP__TIMESTAMP = FunctionUtils.hidden(
```

Comment:
See these, I recommend just adding ".hidden" or ".system" to Function rather than FunctionUtils, but I also don't know if you had to deal with visibility modifier issues.